packages/rtm-api/src/RTMClient.ts

```diff
@@ -561,7 +561,7 @@ export class RTMClient extends EventEmitter {
         this.logger.debug(`sending message on websocket: ${flatMessage}`);
 
         this.websocket.send(flatMessage, (error) => {
-          if (error !== undefined) {
+          if (error) {
```

Comment:
To prevent a regression, what do you think about adding a comment above this conditional stating that Node.js 19.x returns `null` while earlier versions return `undefined`?